priv/repo/migrations/20211105165216_create_community_accounts.exs

```diff
@@ -0,0 +1,17 @@
+defmodule Oli.Repo.Migrations.CreateCommunityAccounts do
+  use Ecto.Migration
+
+  def change do
+    create table(:community_accounts) do
+      add :community_id, references(:communities)
```

Comment:
we should add the rest of the references here such as delete_all, nullify_all etc, if you need help deciding which ping me. (rely on db constraints only is good - no need for ORM to also enforce it)

Reply:
I think I didn't get you @jrissler. Could you elaborate this point a little more?

Reply:
You got exactly what I was saying :) Yes delete_all would be good on all here since when a community, author or user is deleted, so should the community accounts